mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -66,8 +71,10 @@ pub enum Error {
     AlreadyShared { name: String },
     #[snafu(display("The nexus {} has not been shared", name))]
     NotShared { name: String },
-    #[snafu(display("Failed to share nexus {}", name))]
-    ShareNexus { source: NbdError, name: String },
+    #[snafu(display("Failed to share NBD nexus {}", name))]
```

Comment:
nit; "would failed to share nexus over NBD" not be more descriptive. One could read this as a nexus called NBD failed to share.

Reply:
Ok, will fix.

Reply:
Fixed in commit https://github.com/openebs/Mayastor/commit/8fe37c12e6ad29c1c6c7d94da22f9d67a47a026b